libs/src/api/playlist.js

```diff
@@ -206,18 +207,37 @@ class Playlists extends Base {
     }
   }
 
+  /**
+   * Uploads a cover photo for a playlist without updating the actual playlist
+   * @param {number} playlistId
+   * @param {File} coverPhotoFile the file to upload as the cover photo
+   * @return {string} CID of the uploaded cover photo
+   */
+  async uploadPlaylistCoverPhoto (playlistId, coverPhotoFile) {
+    this.REQUIRES(Services.CREATOR_NODE)
+
+    const updatedPlaylistImage = await this.creatorNode.uploadImage(
+      coverPhotoFile,
+      'true', // square, this weirdly has to be a boolean string
```

Comment:
this is really strange because this looks like a bool. what are you seeing with not a string https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/services/creatorNode/CreatorNode.ts#L440